### PR TITLE
Update JWX dependency to v2

### LINKED ISSUE #70: CoRIM need to upgrade dependent  jwx package to a newer version
CoRIM uses jwx package.

https://lestrrat-go/jwx package needs to be upgraded from v.1.2.x to 
https://lestrrat-go/jwx/v2 v2.x to be compatible with veraison/services and other repositories.

### PR DESCRIPTION
**Pull request by thomas-fossati**
Fix #70

Signed-off-by: Thomas Fossati <thomas.fossati@arm.com>

### CODE REVIEW

File: corim/signer.go [hunk 1]
```diff
@@ -4,35 +4,27 @@ import (
...
 	"fmt"
 	"reflect"
 
-	"github.com/lestrrat-go/jwx/jwk"
+	"github.com/lestrrat-go/jwx/v2/jwk"
 	cose "github.com/veraison/go-cose"
 )
 
-func getAlgAndKeyFromJWK(j string) (cose.Algorithm, crypto.Signer, error) {
+func getAlgAndKeyFromJWK(j []byte) (cose.Algorithm, crypto.Signer, error) {
```
Comment (yogeshbdeshpande) [inline corim/signer.go:14]: I did not changed the function but used the equivalent API ks.Key(0) which returns the key on Index 0 and it works, with minimal change!

Comment (thomas-fossati) [reply corim/signer.go:14]: since we don't deal in keysets it looks more correct to change this to use the right api

Comment (yogeshbdeshpande) [reply corim/signer.go:14]: However the earlier implementation was using key set, isn't it?

Anyway either way it is fine, we are using the same library tools!

**Comment by yogeshbdeshpande** [comment]
Thanks for the quick change!

**Pull request merged**